keras_cv/layers/preprocessing/random_resized_crop.py

```diff
@@ -194,6 +187,9 @@ def _check_class_arguments(
     def augment_target(self, augment_target, **kwargs):
         return augment_target
 
+    def augment_segmentation_mask(self, segmentation_mask, transformation, **kwargs):
+        return self._crop_and_resize(segmentation_mask, transformation)
```

Comment:
Will this have the same issue for mixing the pixel value when resize? eg if the [[0, 2], [0, 2]] is resize to [1, 1,]?

Reply:
Yes -- I overlooked the interpolation issue for this layer. I will update this and send for re-review later today.

Reply:
I've updated this implementation to use the build-in nearest-neighbor approach for interpolation, which handles the discrete class issue and avoids bad interpolation + rounding.